docs/language.rst

```diff
@@ -643,6 +643,33 @@ To prevent string interpolation, use raw strings
     motd = r"Welcome to {{hostname}}\n"
 
 
+String formatting
```

Comment:
When this is compiled, we get errors like this. And the titles are not shown in the docs.

```
/home/arnaud/inmanta-core/docs/language.rst:623: CRITICAL: Title level inconsistent:

String interpolation
--------------------
/home/arnaud/inmanta-core/docs/language.rst:646: CRITICAL: Title level inconsistent:

String formatting
-----------------
/home/arnaud/inmanta-core/docs/language.rst:673: CRITICAL: Title level inconsistent:

Templates
---------
```

This is an issue that already existed before you added your changes, but I do think we need to fix this.